components/pagination/collections/paging_collection.js

```diff
@@ -377,6 +373,9 @@
                 }
             }
         });
+
+        var PagingCollectionProto = PagingCollection.prototype;
```

Comment:
I'm all for declaring variables at the top of the function, but this is technically declaring the variable at the "top of the function", since it's the second statement in the function following another variable declaration. I could replace the `var` with the comma, but the indentation just makes it look sort of weird.